### ISSUE TITLE
OS detection fails on Amazon AMI

### ISSUE THREAD

**Issue by anton-ko** [serverscope/serverscope-benchmark#23]

**Comment by anton-ko** [comment]
`platform.dist()` returns `('', '', '')` on Amazon AMI

However, it does respond to `platform.linux_distribution(supported_dists=['system'])` with `('Amazon Linux AMI', '2017.09', '')`.

I'm going to add a check in `serverscope_benchmark/__main__.py`.

```python
    payload["os"] = platform.dist()
    if not payload["os"]  and os.path.isfile('/etc/system-release'):
        payload["os"] = platform.linux_distribution(supported_dists=['system'])
```

**Comment by anton-ko** [comment]
Fixed with 47c875bf6b5bacc1dee949d56b24e48b1a3944f2
